Gemfile

```
@@ -25,10 +26,10 @@ gem "newrelic_rpm"
 gem "nokogiri", ">=1.10.5"
 gem "omniauth-saml-va", git: "https://github.com/department-of-veterans-affairs/omniauth-saml-va", branch: "pek-iam-ssoi"
 #gem "omniauth-saml-va", git: "https://github.com/department-of-veterans-affairs/omniauth-saml-va", ref: "fbe2b878c250b14ee996ef6699c42df2c42e41a1"
-gem "pg", "~> 0.18", platforms: :ruby
+gem "pg", "~> 1.1", platforms: :ruby
```

I'm guessing this this `pg` update was prompted by the following error when you ran `bin/rails app:update`?

```
rails aborted!
LoadError: Error loading the 'postgresql' Active Record adapter. Missing a gem it depends on? can't activate pg (~> 1.1), already activated pg-0.21.0. Make sure all dependencies are added to Gemfile.

Caused by:
Gem::LoadError: can't activate pg (~> 1.1), already activated pg-0.21.0. Make sure all dependencies are added to Gemfile.

Tasks: TOP => railties:install:migrations => db:load_config
```

After much Googling and toying around, I could not find a satisfactory solution that didn't mean upgrading `pg`.

I didn't see any breaking changes in the [`pg` changelog](https://github.com/ged/ruby-pg/blob/master/History.md) on the way to `v1.5.4`, however, I'm wondering if we shouldn't keep this pinned at `1.1.Z` out of caution? 🤔 

```suggestion
gem "pg", "~> 1.1.0", platforms: :ruby
```

**Note:** You'll have to `bundle update pg` to update the `Gemfile.lock` as well.